openmc/mgxs/mgxs.py

```diff
@@ -1806,8 +1819,6 @@ def get_xs(self, in_groups='all', out_groups='all',
                                           filter_bins=filter_bins,
                                           nuclides=query_nuclides, value=value)
 
-        xs = np.nan_to_num(xs)
```

Comment:
Ok that's fine. I was just worried since I thought I saw one or two place where it was removed but not added back in. Must have been mistaken. This `nan_to_num` is pretty important if you generate "noisy" MGXS with few particle histories, and is hence important to my work!